tests/test_objects.py

```diff
@@ -166,37 +163,32 @@ def test_GridFsOperations(self):
         
         try:
             # Tests writing to a new gridfs file
-            gfs = gridfs.GridFS(db) # Default collection
-            g_in = gfs.new_file(filename='optest', contentType="text/plain",
-                            chunk_size=2**2**2**2) # non-default chunk size used
+            gfs = gridfs.GridFS(db)  # Default collection
+            g_in = gfs.new_file(filename="optest", contentType="text/plain",
+                                chunk_size=2**2**2**2)  # non-default chunk size used
             # yielding to ensure writes complete before we close and close before we try to read
             yield g_in.write(in_file.read())
             yield g_in.close()
             
             # Tests reading from an existing gridfs file
-            g_out = yield gfs.get_last_version('optest')
+            g_out = yield gfs.get_last_version("optest")
             data = yield g_out.read()
             out_file.write(data)
             _id = g_out._id
-        except Exception,e:
+        except Exception, e:
```

Comment:
Since this commit is mainly related to coding style, may be its worth to change `except Exception, e:` to `except Exception as e:`?